src/sql/workbench/services/tableDesigner/browser/tableDesignerComponentInput.ts

```diff
@@ -232,7 +232,7 @@ export class TableDesignerComponentInput implements DesignerComponentInput {
 			columnProperties.push(...designerInfo.view.additionalTableColumnProperties);
 		}
 
-		const columnsTableProperties = designerInfo.view.columnsTableProperties || [
+		const columnsTableProperties = designerInfo.view.columnsTableProperties?.length > 0 ? designerInfo.view.columnsTableProperties : [
```

Comment:
empty array: `[]` is considered to be `true` when used as boolean, so I am checking whether it is null and its length